libcloud/compute/types.py

```diff
@@ -236,15 +236,27 @@ class StorageVolumeState(object):
     """
     Standard states of a StorageVolume
     """
+    AVAILABLE = "available"
```

Comment:
True, but I personally find "backup" state a bit confusing (backuping is also better, imo, but snapshotting is more provider agnostic) :)